pkg/kube/resources.go

```diff
@@ -173,6 +199,18 @@ func addResourceFromString(contents string, resources *ResourceProvider) error {
 		dep := appsv1.StatefulSet{}
 		err = decoder.Decode(&dep)
 		resources.StatefulSets = append(resources.StatefulSets, dep)
+	} else if resource.Kind == "DaemonSet" {
+		dep := appsv1.DaemonSet{}
```

Comment:
nit: I clearly copy/pasta'd the code for StatefulSet - `dep` was originally meant to refer to `Deployment`. Maybe change `dep` to `ctrl` everywhere?

Reply:
Ah sounds good. Yeah I guess dep didn't connect for me :) I'll fix that up